### PR TITLE
add tests for Kindle tags and total tag count

### PR DESCRIPTION
**Pull request by gidsg**
CC @guardian/content-platforms @mchv

### CODE REVIEW

File: app/com/gu/contentapi/sanity/KindleTagTest.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  "Tags required by the Kindle app" should "exist" in {
+    handleException{
+
+      val kindleTags = Seq(
+        "theguardian/mainsection",
+        "theguardian/mainsection/topstories",
+        "tone/advertisement-features",
+        "uk/uk",
+        "theobserver/news/worldnews")
```
Comment (mchv) [inline app/com/gu/contentapi/sanity/KindleTagTest.scala]: May you add those ones as well:

 * business/business
 * culture/culture
 * football/football
 * theguardian/mainsection/opinion
 * lifeandstyle/lifeandstyle
 * environment/environment

File: app/com/gu/contentapi/sanity/KindleTagTest.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package com.gu.contentapi.sanity
+
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{Matchers, FlatSpec}
+
+class KindleTagTest extends FlatSpec with Matchers with ScalaFutures with IntegrationPatience {
+
+  "Tags required by the Kindle app" should "exist" in {
```
Comment (mchv) [inline app/com/gu/contentapi/sanity/KindleTagTest.scala]: I will say tags critical for publication (notably kindle app)

**Comment by mchv** [comment]
:+1: Add 2 minor improvements requests :)

**Comment by gidsg** [comment]
OK @mchv I made the suggested changes so will merge now.

**Pull request merged**